### PR TITLE
Supporting custom message on Progress page

### PR DESCRIPTION
**Pull request by diegomillan**
### Description

This PR is supports this [feature](https://github.com/Stanford-Online/cme-theme/pull/5). 
### Reviewers

If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [x] Code review: @stvstnfrd
- [x] Code review: @felipemontoya

FYI: Tag anyone who might be interested in this PR here

@juancamilom
### Post-review
- [x] Rebase and squash commits

### CODE REVIEW

**Comment by felipemontoya** [comment]
:+1: this picks the feedback from the conversation at https://github.com/Stanford-Online/cme-theme/pull/5

File: lms/templates/courseware/progress-info.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<%!
+from django.utils.translation import ugettext as _
+%>
+<section class="progress-info-mouseover">
+    <p>
+        ${_("""Note: Submitted assignments will appear as bars in the Progress chart.
+                Mouse over a bar to see more information about the relevant assignment.
+        """)}
+    </p>
+</section>
```
Comment (caesar2164) [inline lms/templates/courseware/progress-info.html]: trailing whitespace

**Comment by stvstnfrd** [comment]
Github is confused, but I made the newline change myself, amended the commit, and merged it here https://github.com/Stanford-Online/edx-platform/commit/4c2e98996dd0d6bc357eb04511e87615119e1079

**Comment by stvstnfrd** [comment]
This is in master.

**Pull request closed without merging**